internal/controller/drplacementcontrol_controller.go

```diff
@@ -60,7 +60,7 @@ const (
 	DoNotDeletePVCAnnotationVal = "true"
 )
 
-var InitialWaitTimeForDRPCPlacementRule = errorswrapper.New("Waiting for DRPC Placement to produces placement decision")
+var ErrInitialWaitTimeForDRPCPlacementRule = errors.New("waiting for DRPC Placement to produces placement decision")
```

Comment:
While you are at it, remve the `s` from `produces`

Reply:
fixed